src/main/java/racinggame/car/Car.java

```diff
@@ -0,0 +1,37 @@
+package racinggame.car;
+
+public class Car {
+    private int position;
+
+    private Car() {
+        this.position = 1;
+    }
+
+    public static Car of() {
+        return new Car();
+    }
+
+    public void changePosition(int randomNum) {
+        if (isKeep(randomNum)) {
+            return;
+        }
+
+        this.position++;
```

Comment:
```suggestion
    public void changePosition(int randomNum) {
        if (isMove(randomNum)) {
            this.position++;
        }
```
위와 같이 구현하는 것이 더 간결하고 명확하지 않을까?